it/support/stubs/CommonStubs.scala

```diff
@@ -52,6 +57,21 @@ trait CommonStubs {
       ))
   }
 
+  def stubActiveEnrolments(enrolmentKeys: String*) = {
+    val enrolmentsAsJson = enrolmentKeys
+      .map(key => s"""{"key": "$key", "identifiers": [{"key": "$key-id", "value": "${(Random.nextFloat() * 100000).toInt}"}], "state": "Activated"}""")
```

Comment:
the Random.nextFloat could be replaced by generating a Random.nextInt, then you woudn't need the `.toInt`